webapp/index.html

```diff
@@ -156,7 +156,7 @@ <h1 style="text-align:center">Kafka Monitor GUI</h1>
     return charts;
   }
 
-  function updateChartAndValueTable(mbeans, charts, cells) {
+  function updateChartAndValueTable(j4p, mbeans, charts, cells) {
```

Comment:
Do we need the j4p parameter in this method?

Reply:
j4p would no longer be in scope for this method otherwise. If you have a more elegant solution, please let me know ( i'm sure there is .. )

Reply:
Ah I didn't notice that j4p is already used in the function. Sure, it makes sense.